test/base/EscrowFactoryCreateEscrowTest.js

```diff
@@ -26,6 +29,8 @@ contract("EscrowFactoryCreateEscrowTest", function(accounts) {
 
   beforeEach(async () => {
     const CETH = await Mock.new();
+    const loanLib = await LoanLib.new();
```

Comment:
This is painful.

Reply:
Yes, but only solid way to reduce the size of the LoansBase contract, unfortunately.